docker-compose-services/redis/README.md

```diff
@@ -60,27 +60,70 @@ $ ddev describe
 │          │      │ Host: localhost:54710                   │                       │
 ```
 
-## Examples
+This example shows:
 
-Frameworks, such as Laravel, require configuration to use the redis service.
+* From the webserver point-of-view:
+  * REDIS HOST: ddev-example-redis
+  * REDIS PORT: 6379
+
+* From the host Operating System point-of-view:
+  * REDIS HOST: localhost
+  * REDIS PORT: 54710
```

Comment:
The `docker-compose.redis.yml` file should have a reference to internal port number `6379`, yes.

But the port exposed to host machine will likely to differ for each user depending on their Docker usages. 

I don't think it's a good idea to include to port for the host machine without any explanation. Please consider adding an explanation that port number will likely to be different for the user.

Reply:
The idea was to provide an example output and teach users how to read it.  The URL and hostname are also project specific but I think people will be able to follow.